quarkus/runtime/src/main/java/org/keycloak/provider/quarkus/KeycloakConfigSourceProvider.java

```diff
@@ -58,23 +55,16 @@
         }
 
         if (filePath != null) {
-            sources.add(wrap(new KeycloakPropertiesConfigSource.InFileSystem(filePath)));
+            sources.add(new KeycloakPropertiesConfigSource.InFileSystem(filePath));
```

Comment:
I'm doing similar changes in another PR. I'm wondering if we should get first my changes even though we need some more discussion around configuration.

It should not be a problem as most of the changes are necessary.